connect/runtime/src/main/java/org/apache/kafka/connect/runtime/standalone/StandaloneHerder.java

```diff
@@ -237,6 +237,17 @@ private synchronized void putConnectorConfig(String connName,
         }
     }
 
+    @Override
+    public synchronized void stopConnector(String connName, Callback<Void> callback) {
+        try {
+            removeConnectorTasks(connName);
```

Comment:
> I guess this shouldn't really matter anyway because we're publishing an empty set of task configs?

Exactly--and thanks to the logic introduced in the [fix](https://github.com/apache/kafka/pull/8118) for [KAFKA-9472](https://issues.apache.org/jira/browse/KAFKA-9472), once the rebalance caused by the empty set of task configs has completed, the `DistributedHerder` class will [automatically wipe the deleted tasks from the status store](https://github.com/apache/kafka/blob/31440b00f3ed8de65f368d41d6cf2efb07ca4a5c/connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/DistributedHerder.java#L2368).